### ISSUE TITLE
Basic example not working

### ISSUE THREAD

**Issue by knelson-hme** [google-ai-edge/ai-edge-torch#752]
I am trying to get a basic example working that converts a simple pytorch model to a quantized tflite file. I am using python3.12 with pytorch2.7.0 and ai-edge-torch0.5.0 and created the below code. I tried following [this](https://docs.pytorch.org/ao/main/tutorials_source/pt2e_quant_x86_inductor.html) and I also tried following the quantization section located [here](https://github.com/google-ai-edge/ai-edge-torch/tree/main/docs/pytorch_converter), but I would normally end up with an error like `error: 'stablehlo.uniform_dequantize' op operand #0 must be ranked tensor of per-tensor integer quantized or per-axis integer quantized values, but got 'tensor<10x5xi8>'`. There must be something wrong with my quantizer and I show a couple different attempts in the code, but none of them worked. I am just wondering if anyone can duplicate my problem and maybe point out what I am doing wrong. Thanks!

```python
import torch
import torch.ao.quantization
from torch.ao.quantization.quantize_pt2e import prepare_pt2e, convert_pt2e
from ai_edge_torch import convert

from ai_edge_torch.quantize.pt2e_quantizer import get_symmetric_quantization_config
from ai_edge_torch.quantize.pt2e_quantizer import PT2EQuantizer
import torch.ao.quantization.quantizer.x86_inductor_quantizer as xiq
from torch.ao.quantization.quantizer.x86_inductor_quantizer import X86InductorQuantizer
# from executorch.backends.xnnpack.quantizer.xnnpack_quantizer import (
#   get_symmetric_quantization_config,
#   XNNPACKQuantizer,
# )
from ai_edge_torch.quantize.quant_config import QuantConfig


class M(torch.nn.Module):
   def __init__(self):
      super().__init__()
      self.linear = torch.nn.Linear(5, 10)

   def forward(self, x):
      return self.linear(x)


example_inputs = torch.randn(1, 5)
m = M().eval()

m = torch.export.export(m, (example_inputs,)).module()

# Quantizer Attempt1
quantizer = PT2EQuantizer().set_global(
    get_symmetric_quantization_config(is_per_channel=True, is_dynamic=True)
    # get_symmetric_quantization_config()
)

# # Quantizer Attempt2
# quantizer = X86InductorQuantizer()
# quantizer.set_global(xiq.get_default_x86_inductor_quantization_config())


# # Quantizer Attempt3
# quantizer = XNNPACKQuantizer()
# quantizer.set_global(get_symmetric_quantization_config())

m = prepare_pt2e(m, quantizer)

with torch.no_grad():
    m(example_inputs) # Use your actual calibration data here

m = convert_pt2e(m)

edge_model = convert(m, (example_inputs,))
# edge_model = convert(m, (example_inputs,), quant_config=QuantConfig(pt2e_quantizer=quantizer))
edge_model.export("out/Test.tflite")  # Export to TFLite format

print("Done")
```

**Comment by gaikwadrahul8** [comment]
Hi, @knelson-hme 
I apologize for the delay in my response, I am able to reproduce the same behavior from my end with your provided code snippet for reference here is [gist-file](https://colab.sandbox.google.com/gist/gaikwadrahul8/9bce59fb79a3fc6c3011662bd2a36e74/ai-edge-torch-issue-752.ipynb) so if I'm not wrong it's happening because the conversion pipeline expected a fully quantized model (with quantized activations) but the dynamic quantization configuration produced a model with floating-point activations which is not a self-contained, generally usable quantized model. I have tried static quantization which is fully supported and recommended path which is working as expected here is [gist-file](https://colab.sandbox.google.com/gist/gaikwadrahul8/2f6bc5f1f623b99515ab444f85a101f3/ai-edge-torch-issue-752.ipynb) for reference.

https://github.com/google-ai-edge/ai-edge-torch/blob/46ff6d0587b0a86ce4afc56d23cd40b569ef41fe/ai_edge_torch/quantize/pt2e_quantizer.py#L159

If I have missed something here please let me know. Thank for your cooperation and understanding.

**Comment by knelson-hme** [comment]
Thank you for your reply.
I ran your code, but when I look at the output tflite file in [Netron](https://netron.app/) I see that it hasn't been quantized and that the weights are still float. Do you see this same thing? Maybe I am reading the Netron graph wrong or using it incorrectly.

**Comment by gaikwadrahul8** [comment]
Hi, @knelson-hme 
If possible could you please help me with your Netron app output log for quantized model ? Thank you

**Comment by knelson-hme** [comment]
Hello @gaikwadrahul8,
I ran your [colab code](https://colab.research.google.com/gist/gaikwadrahul8/2f6bc5f1f623b99515ab444f85a101f3/ai-edge-torch-issue-752.ipynb) above and downloaded the generated tflite model. I then put it into [Netron](https://netron.app/) and got the below graph. I clicked on the weights and it shows the type as float32. I expected this to be an integer of some kind like int8. Is that a correct expectation?
<img width="586" height="434" alt="Image" src="https://github.com/user-attachments/assets/fbc8611e-6176-4489-96d7-545c661c1f56" />

**Comment by aravindmurali22** [comment]
Hi @rewu93 , following up on this thread, is there any update on this issue? Thanks!
